internal/webhook/external/github.go

```diff
@@ -71,83 +80,74 @@ func githubHandler(
 			return
 		}
 
+		logger = logger.WithValues("eventType", eventType)
+		ctx = logging.ContextWithLogger(ctx, logger)
+
 		const maxBytes = 2 << 20 // 2MB
```

Comment:
If `token` becomes a `const`, it might be worth moving this into that block as well (instead of within the method).